PowerEditor/src/Notepad_plus.cpp

```diff
@@ -1351,46 +1351,29 @@ void Notepad_plus::removeEmptyLine(bool isBlankContained)
 	_findReplaceDlg.processAll(ProcessReplaceAll, &env, true);
 }
 
-void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir)
+void Notepad_plus::getMatchedFileNames(const TCHAR *dir, const vector<generic_string> & patterns, vector<generic_string> & fileNames, bool isRecursive, bool isInHiddenDir, const Progress* progress)

```

Comment:
@ariccio Overlooking the disturbing amount of political incorrectness of your analogies, it seems you recognize your perspective is flawed yet you continue to justify your suggestions for your own personal preferences. There is no general consensus that agrees with your preference of const for by-value parameters. In any case I'm at fault for something here too, and that is the continuation of and contribution to a conversation which has run its course on a topic that isn't very important to begin with. I'm sure there are far more valuable discussion we can have on this PR without wasting everyone's time. I apologize for becoming involved in the discussion, I'll try to avoid this topic next time :)